src/App.tsx

```diff
@@ -1,156 +1,119 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { Header } from './components/Header';
+import { Main } from './components/Main';
+import { Footer } from './components/Footer';
+import { Todo } from './types/Todo';
+import { getTodos } from './api/todos';
 
-const USER_ID = 0;
+const USER_ID = 11359;
 
 export const App: React.FC = () => {
+  const [myTodos, setMyTodos] = useState<Todo[]>([]);
+  const [closeError, setCloseError] = useState(false);
+  const [errorMassege, setErrorMassege] = useState('');
+  const [query, setQuery] = useState('All');
+  const [isCompleted, setIsCompleted] = useState(false);
+  const [numberActive, setNumberActive] = useState(0);
+
+  function hideError() {
+    setTimeout(() => setCloseError(true), 3000);
```

Comment:
you can just `setErrorMassege('')` - empty string is falsy value
and then!!errorMessage will always be the same as `closeError` now - so you can remove this state